messaging/rabbitmq_binding.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module: rabbitmq_binding
+author: Manuel Sousa
+version_added: 2.0
```

Comment:
`version_added` value must be quoted because it is not a float number, so add change it to `"2.0"`